src/polyswarm_api/__init__.py

```diff
@@ -923,27 +936,23 @@ def scan_directory(self, directory, recursive=False):
         """
         return self.loop.run_until_complete(self.ps_api.scan_directory(directory, recursive))
 
-    def search_hashes(self, hashes, hash_type='sha256'):
+    def search_hashes(self, hashes):
```

Comment:
we want to force the user to specify the type of hash. And if they do not specify the type of hash then we will assume sha256. So, I need you to re-add the "hash_type" arg to all of these functions. Then update the docs to tell the user to specify the hash type on the command line. And if no hash type is provided, then we will assume sha256.